packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -361,6 +386,25 @@ export class TaskDefinition extends TaskDefinitionBase {
    * Adds a new container to the task definition.
    */
   public addContainer(id: string, props: ContainerDefinitionOptions) {
+    if (this.cpu !== undefined
+          && props.cpu !== undefined
+          && parseInt(this.cpu, 10) < props.cpu) {
```

Comment:
Sooo... no suffix then? Then why is it a string?

You need to check for either `this.cpu` or `props.cpu` to be tokens.

Reply:
It should have been a number, we'll change it when breaking changes are allowed.